pytextnow/TNAPI.py

```diff
@@ -7,6 +7,7 @@
     from pytextnow.multi_media_message import MultiMediaMessage
     from pytextnow.message import Message
 import mimetypes
+from re import M

```

Comment:
Was this for something I changed?

Reply:
Oh no my editor must of added that automatically

Reply:
alright is there anything else I mightve missed and ill group it all into one commit